src/app/compare/compare-buttons.tsx

```diff
@@ -36,15 +36,15 @@ export function findSimilarArmors(exampleItem: DimItem): CompareButton[] {
     {
       buttonLabel: [
         <ArmorSlotIcon key="slot" item={exampleItem} className={styles.svgIcon} />,
-        `+ ${t('Stats.Sunset')}`,
+        `+ ${t('Compare.Armor2')}`,
       ],
       query: '', // since we already filter by itemCategoryHash, an empty query gives you all items matching that category
     },
 
     // above but also has to be armor 2.0
     exampleItem.destinyVersion === 2 && {
       buttonLabel: [<ArmorSlotIcon key="slot" item={exampleItem} className={styles.svgIcon} />],
-      query: 'not:sunset',
+      query: 'is:armor2.0',
```

Comment:
This aligns the search with the comment, and I think the intention?

Reply:
correct about armor2 and sunset being nearly equal. but aren't we in armor 2.0 now? the reason "Sunset" went on the button is because "+ Armor <2.0" is confusing/silly looking

Reply:
so concept aligned, but the button needs to express "includes armor that isn't reusable-mods-based"

Reply:
I updated the wording to "Pre-mods", what do you think?